cypress/e2e/audio-player/audio-player.spec.ts

```diff
@@ -0,0 +1,563 @@
+                // Assert that media information is rendered
+                cy.get(".mx_AudioPlayer_mediaInfo").within(() => {
+                    cy.get(".mx_AudioPlayer_mediaName").should("have.text", "1sec-long-name-audio-file.ogg");
+                    cy.contains(".mx_AudioPlayer_byline", "00:01").should("exist");
+                    cy.contains(".mx_AudioPlayer_byline", "(3.56 KB)").should("exist"); // actual size
+                });
+
+                // Assert that the play button is rendered
+                cy.get("[data-testid='play-pause-button'][aria-label='Play']").should("exist");
+            });
+        });
+    };
+
+    // Take snapshots in modern and bubble layout, outputting log for reference/debugging
+    // Player on IRC layout should have the same layout as group layout
+    const takeSnapshots = (wrapper: string, detail: string) => {
+        // Check the status of the seek bar
+        // TODO: check if visible - currently visibility check on a narrow timeline causes an error
+        cy.get(`${wrapper}.mx_AudioPlayer_seek input[type='range']`).should("exist");
+
+        // Assert that the pause button is not rendered
+        cy.get(`${wrapper}[data-testid='play-pause-button'][aria-label='Pause']`).should("not.exist");
+
+        // Assert that the play button is rendered
+        cy.get(`${wrapper}[data-testid='play-pause-button'][aria-label='Play']`).should("exist");
+
+        // Enable group layout
+        cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Group);
+
+        // 243 + 12px + 12px = 267px
+        // See _MediaBody.pcss and _AudioPlayer.pcss for spacing
+        cy.get(`${wrapper}.mx_MAudioBody`).percySnapshotElement(detail + " on group layout", {
+            percyCSS,
+            widths: [267],
+        });
+
+        cy.log("Took a snapshot of " + detail + " on group layout");
+
+        // Enable bubble layout
+        cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Bubble);
+
+        // 243px + 12px + 48px = 303px
+        // See _EventBubbleTile.pcss and _AudioPlayer.pcss for spacing
+        cy.get(`${wrapper}.mx_MAudioBody`).percySnapshotElement(detail + " on bubble layout", {
+            percyCSS,
+            widths: [303],
+        });
+
+        cy.log("Took a snapshot of " + detail + " on bubble layout");
+
+        // Reset the layout
+        cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Group);
+    };
+
+    beforeEach(() => {
+        cy.startHomeserver("default").then((data) => {
+            homeserver = data;
+            cy.initTestUser(homeserver, TEST_USER).then(() =>
+                cy.createRoom({}).then((_roomId) => {
+                    roomId = _roomId;
+                }),
+            );
+        });
+
+        cy.injectAxe();
+    });
+
+    afterEach(() => {
+        cy.stopHomeserver(homeserver);
+    });
+
+    it("should render player on every layout", () => {
```

Comment:
we already have "Audio player" in `describe`, so no need to repeat "player" here.

```suggestion
    it("should be correctly rendered on each layout", () => {
```